buildlib/pr/io_demo/io-demo.yml

```diff
@@ -35,6 +35,22 @@ parameters:
         interface: $(roce_iface_cx6)
         tls: "rc_x"
         test_name: tag_umemh_cx6_rc
+      "server one path compatible on CX4":
+        args: ""
+        initial_delay: 9999 # No interference
+        duration: 600
```

Comment:
sorry, one more: please reduce the test duration to 120, since just need to test connection establishment and no interference. can force-push the fix.